src/plugins/initiatives/initiativesMap.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+import {type Compatible, fromCompat, toCompat} from "../../util/compat";
+import {type URL} from "./initiative";
+
+const COMPAT_INFO = {type: "sourcecred/initiatives", version: "0.1.0"};
+
+/**
+ * JSON (compat) representation of an Initiative.
+ *
+ * Major difference is that instead of using tracker addresses, we use object
+ * keys as an ID, and derive the tracker address from that.
+ */
+export type InitiativesMap = {[entryKey: string]: InitiativeEntry};
+
+type InitiativeEntry = {|
+  +title: string,
+  +timestampMs: number,
+  +completed: boolean,
+  +dependencies: $ReadOnlyArray<URL>,
+  +references: $ReadOnlyArray<URL>,
+  +contributions: $ReadOnlyArray<URL>,
+  +champions: $ReadOnlyArray<URL>,
```

Comment:
I think it should be `$ReadOnlyArray<Alias>` instead of URL. Since we already have a concise sublanguage that's intended for describing user identities.

Reply:
These types are compatible with the main `Initiative` type, which we will need for conversion.
https://github.com/sourcecred/sourcecred/pull/1417/files

Using aliases would increase complexity, as we would need two types of resolvers. One for aliases and another for URLs.